WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -274,8 +278,24 @@ private extension ShippingLabelAddressFormViewController {
         cell.configure(viewModel: cellViewModel)
     }
 
-    func configureFieldError(cell: BasicTableViewCell, row: Row) {
-        cell.textLabel?.text = viewModel.addressValidationError?.addressError
+    func configureFieldError(cell: BasicTableViewCell, row: Row, error: ShippingLabelAddressFormViewModel.ValidationError) {
+        var errorMessage = viewModel.addressValidationError?.addressError
+        switch error {
+        case .name:
+            errorMessage = Localization.invalidName
+        case .address:
+            errorMessage = viewModel.addressValidationError?.addressError ?? Localization.invalidAddress
+        case .city:
+            errorMessage = Localization.invalidCity
+        case .postcode:
+            errorMessage = Localization.invalidPostcode
+        case .state:
+            errorMessage = Localization.invalidState
+        case .country:
+            errorMessage = Localization.invalidCountry
```

Comment:
I think these names are a bit misleading. Do you think it makes sense to have something like "Missing {attribute}" instead of "Invalid {attribute}"